src/components/icon-button/icon-button.tsx

```diff
@@ -99,12 +62,6 @@ export class IconButton {
             buttonAttributes.tabindex = this.host.getAttribute('tabindex');
         }
 
-        const iconAttributes: { badge?: boolean; size?: IconSize } = {};
-        if (this.elevated) {
-            iconAttributes.badge = true;
-            iconAttributes.size = 'small';
```

Comment:
![Screenshot 2022-07-12 at 14 58 32](https://user-images.githubusercontent.com/35954987/178511245-a67d32e4-a6b3-41ff-b018-f09a8cfd5b1a.png)

icon size is now handled in the styles instead, and badge is true always. So background color can be applied to the entire buttons (not just the icon part), using the same old CSS var of `--icon-background-color`.

Reply:
So, the button is visually slightly larger, but actually takes less space in the page flow? That's nice 👍